### PR TITLE
chore: update check manifest content

### PR DESCRIPTION
**Pull request by xiaolang124**
1.Update unify config manifest readme.
2.Add learn more to notify user to update manifest.

image

### CODE REVIEW

File: packages/fx-core/resource/unify-config-change-logs.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Your existing `localSettings.json`, `manifest.local.template.json`, and `manifes
 ## Know what you need to do
 Since Teams Toolkit will use `manifest.remote.template.json` as a single manifest template file after the upgrade, if you have customized the `manifest.local.template.json`, you will need to update the `manifest.template.json` to include your change.
 
+For more information you can visit [Unify config help wiki](https://github.com/OfficeDev/TeamsFx/wiki/Upgrade-to-support-unify-config).
```
Comment (kimizhu) [inline packages/fx-core/resource/unify-config-change-logs.md:22]: use aka link, we can link to anchor of the page later.

File: packages/fx-core/resource/package.nls.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
...
   "core.consolidateLocalRemote.Message": "Your project will be upgraded to work with the latest version of Teams Toolkit. The upgrade will only backup and change the local config and manifest template file without changing your custom code. If you want to skip auto-upgrade, please install older version of Teams Toolkit (<=3.7.0)",
-  "core.consolidateLocalRemote.DifferentManifest": "Since Teams Toolkit will use manifest.remote.template.json as a single manifest template file after the upgrade, if you have customized the manifest.local.template.json, you will need to update the manifest.template.json to include your change.",
+  "core.consolidateLocalRemote.DifferentManifest": "Since Teams Toolkit will use manifest.remote.template.json as a single manifest template file after the upgrade and we detected you may have customized the manifest.local.template.json, you may need to update the manifest.template.json to include your change. See Learn More for detail.",
```
Comment (kimizhu) [inline packages/fx-core/resource/package.nls.json:50]: for details.

File: packages/fx-core/src/core/middleware/consolidateLocalRemote.ts [hunk 3]
```diff
@@ -425,6 +420,20 @@ async function compareLocalAndRemoteManifest(
...
+  const res = await TOOLS?.ui.showMessage(
+    "warn",
+    getLocalizedString("core.consolidateLocalRemote.DifferentManifest"),
+    false,
+    "OK",
+    LearnMore
+  );
+  const answer = res?.isOk() ? res.value : undefined;
+  if (answer === LearnMore) {
+    TOOLS?.ui.openUrl(LearnMoreLink);
```
Comment (kimizhu) [inline packages/fx-core/src/core/middleware/consolidateLocalRemote.ts:433]: use different aka link, we'd better to direct users to one anchor of the wiki page.

File: packages/fx-core/resource/unify-config-change-logs.md (hunk 1 above)
Comment (kimizhu) [inline packages/fx-core/resource/unify-config-change-logs.md:22]: For how to update the `manifest.template.json`, you can check this [wiki page].

Comment (xiaolang124) [reply packages/fx-core/resource/unify-config-change-logs.md]: Fixed.

File: packages/fx-core/resource/package.nls.json (hunk 2 above)
Comment (xiaolang124) [reply packages/fx-core/resource/package.nls.json]: Fixed.

File: packages/fx-core/src/core/middleware/consolidateLocalRemote.ts (hunk 3 above)
Comment (xiaolang124) [reply packages/fx-core/src/core/middleware/consolidateLocalRemote.ts]: Fixed.

File: packages/fx-core/resource/unify-config-change-logs.md (hunk 1 above)
Comment (xiaolang124) [reply packages/fx-core/resource/unify-config-change-logs.md]: Fixed.

**Comment by wenytang-ms** [comment]
please provide the information which tester need to check, for example, issue link ?

File: packages/fx-core/resource/unify-config-change-logs.md [hunk 4]
```diff
@@ -19,6 +19,8 @@ Your existing `localSettings.json`, `manifest.local.template.json`, and `manifes
 ## Know what you need to do
 Since Teams Toolkit will use `manifest.remote.template.json` as a single manifest template file after the upgrade, if you have customized the `manifest.local.template.json`, you will need to update the `manifest.template.json` to include your change.
 
+For how to update the `manifest.template.json`, you can check this [Wiki Page](https://aka.ms/teamsfx-unify-local-remote-manifest-guide).
```
Comment (kimizhu) [inline packages/fx-core/resource/unify-config-change-logs.md:22]: Is the link open issue fixed?

Comment (xiaolang124) [reply packages/fx-core/resource/unify-config-change-logs.md:22]: Yes, it's my local environment problem. I can open this Wiki now.

**Comment by xiaolang124** [comment]
> please provide the information which tester need to check, for example, issue link ?

This pr is doc and UI update for https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/14188744/

**Review by nliu-ms — APPROVED** [review]

**Review by kimizhu — APPROVED** [review]

**Pull request merged**